src/main/java/me/totalfreedom/totalfreedommod/util/FUtil.java

```diff
@@ -36,9 +37,12 @@
     //
     public static final String SAVED_FLAGS_FILENAME = "savedflags.dat";
     // See https://github.com/TotalFreedom/License - None of the listed names may be removed.
-    public static final List<String> DEVELOPERS = Arrays.asList("Madgeek1450", "Prozza", "Wild1145", "WickedGamingUK", "aggelosQQ");
+    public static final List<String> DEVELOPERS = Arrays.asList("Madgeek1450", "Prozza", "WickedGamingUK", "aggelosQQ" ,"Wild1145" , "OxLemonxO", "marcocorriero", "Commodore64x");
+    public static final List<String> FOUNDER = Arrays.asList("markbyron");
```

Comment:
No. It should not. The founder will be different from server to server. I am the founder of CJFreedom, not markbyron. So why should Mark show as the founder on my server when I was the founder? Thats my point here. Move it to a config like the owner is.